demos/total_radiation.py

```diff
@@ -1,28 +1,49 @@
 
+import numpy as np
 import matplotlib.pyplot as plt
-plt.ion()
 
 from cherab.core.atomic import neon
+from cherab.core.math import sample2d_grid
 from cherab.adas import ADAS
 
 
-print("testing total radiation for Neon")
+print("Testing total radiation for Neon.")
 
 atomic_data = ADAS()
 
-plt.figure()
+electron_density = np.array([1.e19])
+electron_temperature = np.logspace(0, 4, 41)
 
-ne_total_rad = atomic_data.radiated_power_rate(neon, 'total')
-ne_total_rad.plot_temperature()
+# Obtaining total radiation using ADAS405 code (equilibrium balance)
+total_rad = atomic_data.total_radiated_power(neon)
+total_power = sample2d_grid(total_rad, electron_density, electron_temperature).squeeze()
 
-ne_line_rad = atomic_data.radiated_power_rate(neon, 'line')
-ne_line_rad.plot_temperature()
+# Obtaining fractional abundance using ADAS405 code
+fractional_abundance = np.zeros((11, electron_temperature.size))
+for charge in range(11):
```

Comment:
Try to avoid hard-coding `11` both times here: it's easy to update one and forget the other at a later time. Maybe `for charge in range(fractional_abundance.shape[0])`. Or just set a variable `num_charges = 11` and use that in the `np.zeros` and `range` calls.

You could also do the following, since iterating over a 2D numpy array returns views of the rows, but it may be overcomplicating things for a simple demo.
```
for charge, abundance in enumerate(fractional_abundance):
    fraction = atomic_data.fractional_abundance(neon, charge)
    abundance[:] = sample2d_grid(fraction, electron_density, electron_temperature).squeeze()
```

Reply:
Done.